### PR TITLE
feat: ✨ add AsciiDoc support (Asciidoctor icon)

### PR DESCRIPTION
**Pull request by Naveandice**
👋 Hello again, I added [AsciiDoc](https://asciidoc.org) support. The AsciiDoc logo is copyrighted by Eclipse (and I don't wanna read their long legal papers about branding), so I replaced it with the [Asciidoctor](https://asciidoctor.org) logo. It doesn't look as good as I wanted, though.

### CODE REVIEW

**Comment by ItsPi3141** [comment]
Please take a look at the icon color guidelines [here](https://github.com/leonardssh/vscord/blob/main/ADDING_LANGUAGE.md#color). I would revert commit bb028d5ad4cdd4c207b1120a6f704b2222d5f085.

**Comment by Naveandice** [comment]
It's hard to follow any guidelines about colors when it's all about different monitors displaying different colors.

I've got back the ea23b0c icon and changed it's background HSL to 342deg 25% 18%. Looking for suggestions.

<details><summary>Preview</summary>

asciidoc

</details>
